source/jormungandr/jormungandr/instance_manager.py

```diff
@@ -349,7 +350,8 @@ def get_instances(self, name=None, lon=None, lat=None, object_id=None, api='ALL'
         valid_instances = self._filter_authorized_instances(available_instances, api)
         if available_instances and not valid_instances:
             # user doesn't have access to any of the instances
-            authentication.abort_request(user=authentication.get_user())
+            context = 'user does not have access to any of the instances'
```

Comment:
```suggestion
            context = 'User does not have access to any of the instances'
```

(only for homogeneity with the rest).